### ISSUE TITLE
errored out during installation

### ISSUE THREAD

**Issue by danielwuz** [thusoy/grunt-nose#12]
Hi, 
I'm having hard time installing `grunt-nose` from npm. Below is the command I ran in command line:

```
$ npm install grunt-nose
```

and I'm receiving error in `postinstall`:

```
> grunt-nose@1.0.0 postinstall /Users/Shared/workspace/kennebec/node_modules/grunt-nose
> cd tasks/lib && pip install nose-1.3.4.tar.gz --target .

sh: line 0: cd: tasks/lib: No such file or directory
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "grunt-nose"
npm ERR! node v4.1.0
npm ERR! npm  v2.14.3
npm ERR! code ELIFECYCLE

npm ERR! grunt-nose@1.0.0 postinstall: `cd tasks/lib && pip install nose-1.3.4.tar.gz --target .`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the grunt-nose@1.0.0 postinstall script 'cd tasks/lib && pip install nose-1.3.4.tar.gz --target .'.
```

my npm and node versions are like below:
```
npm: 2.14.3
node: v4.1.0
```

I'm new to nodejs and grunt, I apologize if I'm missing something basic here, googled solution for an hour and couldn't find any solution. It'd be great if you can help me out.

Thanks,

**Comment by thusoy** [comment]
Are you trying to install the plugin as root? Normally you'd run `npm install grunt-nose --save-dev` in the directory where you have your project, and it will install itself in that directory.

I don't understand why the failure occurs, but you should be able to work around it by splitting the install into two steps. Try the following:

```
npm install --ignore-scripts grunt-nose
cd node_modules/grunt-nose
npm run-script postinstall
cd -
```

**Comment by danielwuz** [comment]
Thanks for your quick reply.

I install the plugin in a virtual environment using [pyenv](https://github.com/yyuu/pyenv). Just tried your suggestion, installing without scripts and manually run them later works for me.

After investigating for a while, I was also able to find a solution, as changes I made in this commit: https://github.com/danielwuz/grunt-nose/commit/7e25f14ec40dc6fb9638c76155901ed25f4d560f 

Then I ran command
```
npm install </my/local/grunt-nose/fork/folder>
```

It also works.

Not sure if this is the right fix, therefore I'll just keep it on my fork.
